eventlet/patcher.py

```diff
@@ -229,7 +231,6 @@ def monkey_patch(**on):
 
     It's safe to call monkey_patch multiple times.
     """
-
```

Comment:
This removal (and and `#_os = original('os')` line above) would be better left out of this PR.

Reply:
Yes, it even broke pep8 checker. Sorry, wasn't careful after trying random stuff with testing.